### ISSUE TITLE
x64 staxrip shipped with x86 DGDecode.dll, error breaks to front end. Solution available

### ISSUE THREAD

**Issue by agentrfr** [staxrip/staxrip#883]
Input file: .mpg MPEG-PS container with MPEG V - 2 video and PCM signed big endian audio

Fix: in StaxRip-v2.13.0-x64\Apps\Support\DgMpgDec replace x86 DGDecode.dll with x64 DGDecode.dll available from https://www.rationalqm.us/dgmpgdec/dgmpgdec2007.zip

```
------------------------- System Environment -------------------------

StaxRip    : v2.13.0
Windows    : Windows 10 Pro 2009
Language   : English (United States)
CPU        : Intel(R) Core(TM) i5-3210M CPU @ 2.50GHz
GPU        : Intel(R) HD Graphics 4000, AMD Radeon HD 7600M Series
Resolution : 1920 x 1080
DPI        : 96
Code Page  : 1252

----------------------- Media Info Source File -----------------------

E:\convert\Conflict Management.mpg

General
Complete name             : E:\convert\Conflict Management.mpg
Format                    : MPEG-PS
File size                 : 928 MiB
Duration                  : 13 min 35 s
Overall bit rate mode     : Variable
Overall bit rate          : 9 541 kb/s

Video
ID                        : 224 (0xE0)
Format                    : MPEG Video
Format version            : Version 2
Format profile            : Main@Main
Format settings           : BVOP
Format, BVOP              : Yes
Format, Matrix            : Default
Format, GOP               : M=3, N=15
Format, picture structure : Frame
Duration                  : 13 min 35 s
Bit rate mode             : Variable
Bit rate                  : 7 815 kb/s
Maximum bit rate          : 8 300 kb/s
Width                     : 720 pixels
Height                    : 576 pixels
Display aspect ratio      : 4:3
Frame rate                : 25.000 FPS
Standard                  : PAL
Color space               : YUV
Chroma subsampling        : 4:2:0
Bit depth                 : 8 bits
Scan type                 : Interlaced
Scan order                : Top Field First
Compression mode          : Lossy
Bits/(Pixel*Frame)        : 0.754
Time code of first frame  : 00:00:00:11
Time code source          : Group of pictures header
Stream size               : 760 MiB (82%)

Audio
ID                        : 189 (0xBD)-160 (0xA0)
Format                    : PCM
Format settings           : Big / Signed
Muxing mode               : DVD-Video
Duration                  : 13 min 35 s
Bit rate mode             : Constant
Bit rate                  : 1 536 kb/s
Channel(s)                : 2 channels
Sampling rate             : 48.0 kHz
Bit depth                 : 16 bits
Delay relative to video   : -342 ms
Stream size               : 149 MiB (16%)

-------------------- DGIndex: Demux & Index MPEG-2 --------------------

DGIndex 2.0.0.7

E:\StaxRip-v2.13.0-x64\Apps\Support\DgMpgDec\DGIndex.exe -i "E:\convert\Conflict Management.mpg" -ia 2 -fo 0 -yr 1 -tn 1 -om 2 -drc 2 -dsd 0 -dsa 0 -o "E:\convert\Conflict Management_temp\Conflict Management" -hide -exit

Start:    10:37:30 AM
End:      10:38:08 AM
Duration: 00:00:38

------------------------ Error opening source ------------------------

Cannot load a 32 bit DLL in 64 bit Avisynth: 'E:/StaxRip-v2.13.0-x64/Apps/Support/DgMpgDec/DGDecode.dll'.

(E:\convert\Conflict Management_temp\Conflict Management_source.avs, line 2)

AddAutoloadDir("E:\StaxRip-v2.13.0-x64\Apps\FrameServer\AviSynth\plugins")
LoadPlugin("E:\StaxRip-v2.13.0-x64\Apps\Support\DgMpgDec\DGDecode.dll")
MPEG2Source("E:\convert\Conflict Management_temp\Conflict Management.d2v")

```

**Comment by fredlkrue** [comment]
Duplicate of:
https://github.com/staxrip/staxrip/issues/881

**Comment by Dendraspis** [comment]
Already fixed in `v2.17.0` .... sorry for the late response.
